client/slack.go

```diff
@@ -147,6 +147,20 @@ func (sl *slackClient) handleMessageEvents() {
 				continue
 			}
 			sl.incomingMessages <- toFlyteMessageEvent(v, u)
+
+		case *slack.ReactionAddedEvent:
+			u, err := sl.client.GetUserInfo(v.User)
```

Comment:
try to be consistent in naming e.g. you are using `itemuser` and for this one you are using just `u` either stick to shorthand in both or expanded names

Reply:
yes, agreed.